### PR TITLE
Refactor maintainer data and types into own file

### PR DESCRIPTION
**Pull request by EmmaDawsonDev**
## Describe your changes
Moved maintainer data into it's own file and created a new IMaintainer type which can be used in both About page and Maintainer Card component. Renamed maintainerData to currentMaintainerData in case we have maintainers that leave in the future then a pastMaintainderData array can be added too.

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave
      and it shows no errors.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/201"></a>

### CODE REVIEW

File: data/maintainers.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export interface IMaintainer {
```
Comment (ctoffanin) [inline data/maintainers.ts:1]: The name `IMaintainer` stands for `Interface Maintainer` or something similar, right?

**Review by ctoffanin — APPROVED** [review]
Nice! Much better.

**Comment by ctoffanin** [comment]
A question. We are using `alt={fullName}` for the image. 
To make a more descriptive `ALT` text. Isn't it better to have an `imageAlt` property in the maintainer data object, where we can be more accurate? (i.e. "Maintainer x smiling in front of a dark background")

File: data/maintainers.ts (hunk 1 above)
Comment (EmmaDawsonDev) [reply data/maintainers.ts:1]: Yes, I was thinking we should probably update it so all interfaces are consistent

**Comment by EmmaDawsonDev** [comment]
> A question. We are using `alt={fullName}` for the image.
> To make a more descriptive `alt` text. Isn't it better to have an `imageAlt` property in the maintainer data object, where we can be more accurate? (i.e. "Maintainer x smiling in front of a dark background")

Hmm, I was wondering about this but I’m not sure how much detail is really needed. But definitely something to consider.

**Pull request merged**